videostore/videostore.go

```diff
@@ -15,11 +14,12 @@ import (
 )
 
 const (
-	defaultSegmentSeconds      = 30 // seconds
-	defaultUploadPath          = ".viam/capture/video-upload"
-	defaultStoragePath         = ".viam/video-storage-viamrtsp"
-	maxGRPCSize                = 1024 * 1024 * 32 // bytes
-	videoStoreInitCloseTimeout = time.Second * 10
+	maxGRPCSize         = 1024 * 1024 * 32 // bytes
+	defaultFramerate    = 20               // frames per second
+	defaultStoragePath  = ".viam/video-storage"
+	defaultUploadPath   = ".viam/capture/video-upload"
```

Comment:
note we are now storing video in the same place the old video-store module did

Reply:
oh i missed this during my review pass